### PR TITLE
Adds support for ANGLE_InstancedArrays extension

### PR DESCRIPTION
**Pull request by jtorresfabra**
Updated the previows PR, but it was better to just do a new one instead of rebasing.. 
Also executed a grunt prettier on the codebase

### CODE REVIEW

**Comment by cedricpinson** [comment]
could you splits commits about prettier and the real changes plz ?

File: sources/osg/Geometry.js [hunk 1]
```diff
@@ -120,6 +122,15 @@ utils.createPrototypeNode(
             }
         },
 
+        setAttribArrayDivisor: function(attributeName, divisor) {
+            if (this._attributes[attributeName] === undefined) {
+                notify.error('Attribute' + attributeName + 'does not exist.');
+                return;
+            }
+            if (this._instancedArrayMap === undefined) this._instancedArrayMap = new window.Map();
```
Comment (jtorresfabra) [inline sources/osg/Geometry.js]: Maybe better to use a {}?

**Comment by jtorresfabra** [comment]
@cedricpinson done

File: sources/osgDB/readerParser.js [hunk 2]
```diff
@@ -204,8 +204,8 @@ ReaderParser.parseSceneGraphDeprecated = function(node) {
                 geom.getPrimitiveSetList().push(new DrawElements(mode, array));
             } else {
                 mode = primitiveSet[mode];
-                var first = primitives[p].first;
-                var count = primitives[p].count;
+                var first = primitives[p].first || primitives[p]._first;
+                var count = primitives[p].count || primitives[p]._count;
```
Comment (Kuranes) [inline sources/osgDB/readerParser.js]: same ?

File: sources/osg/Node.js [hunk 3]
```diff
@@ -36,6 +36,8 @@ var Node = function() {
 
     // it's a tmp object for internal use, do not use
     this._tmpBox = new BoundingBox();
+    this._computeBoundingBoxCallback = undefined;
+    this._computeBoundingSphereCallback = undefined;
```
Comment (Kuranes) [inline sources/osg/Node.js]: _computeBoundingSphereCallback never used ?
need to remove it

**Review by Kuranes — CHANGES_REQUESTED** [review]
- inconsistency between primitives member names (with or without '_') and accessor usage (getIndices())
- doesn't work in webgl2
- Why we do not use the current mechanism in webglcaps to create a backport of webgl2 "gl.DrawElementInstanced, gl.DrawArrayInstanced" to webgl1 ? ( see https://github.com/cedricpinson/osgjs/blob/master/sources/osg/WebGLCaps.js#L365-L383 && https://developer.mozilla.org/en-US/docs/Web/API/ANGLE_instanced_arrays for webgl2 names)

Would give Clearer code and would work on webgl1/2 (and allow to remove the "ANGLE" text from everywhere for just "instanced")

**Pull request closed without merging**